src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -5,7 +5,11 @@ import uniqueId from 'lodash/uniqueId';
 import noop from 'lodash/noop';
 
 import { KEYS } from '../../constants';
+import ResponsiveMenuContainer from './ResponsiveMenuContainer';
```

Comment:
Containers should wrap the components they contain rather than be included as children. Do we need a new DropdownMenu wrapper component?

Reply:
This actually wraps the Menu component that is passed in as a child to the DropdownMenu. Is that acceptable?